### ISSUE TITLE
USB CDC ACM "ring buffer full" 

### ISSUE THREAD

**Issue by xudongzheng** [zmkfirmware/zmk#1832]
Currently ZMK can end up with a ton of `<inf> usb_cdc_acm: Ring buffer full, drain buffer` messages despite it being turned off: https://github.com/zmkfirmware/zmk/blob/19d883cdfe679977023116dcf689ce67a6ce548d/app/Kconfig#L433-L435

This appears to be an issue with Zephyr 3.2 since the underlying `LOG_LEVEL` doesn't work properly: https://github.com/zephyrproject-rtos/zephyr/blob/6130417212a1adefc925851f797f3300fd29fb63/subsys/usb/device/class/cdc_acm.c#L58

This appears fixed in Zephyr 3.4 RC2 (not sure about Zephyr 3.3). A temporary workaround is defining `CONFIG_USB_CDC_ACM_LOG_LEVEL_WRN=y`.

"Hello world" example for reproducing Zephyr issue:

```
#define LOG_LEVEL 0
#include <zephyr/logging/log.h>
LOG_MODULE_REGISTER(test);

void main(void) {
	LOG_INF("Hello World! %s", CONFIG_BOARD);
}
```

**Comment by caksoylar** [comment]
In that case would it make sense to `select USB_CDC_ACM_LOG_LEVEL_WRN` under `ZMK_LOGGING` until the upgrade, perhaps?

**Comment by xudongzheng** [comment]
Closing as this is resolved with the Zephyr 3.5 upgrade.
